microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -45,11 +45,12 @@ Today during the preview, the Evergreen WebView2 Runtime and Microsoft Edge Dev
 > [!IMPORTANT]
 > Do not ship WebView2 applications in production during the preview.  
 
-Please use the following workflow to ensure the Evergreen WebView2 Runtime is available.  
+Applications are recommended to ensure that the Evergreen WebView2 Runtime is installed before launch. Below is an example workflow.  
 
 1.  Download the latest [Evergreen WebView2 Runtime Installer][Webview2Installer].  
 1.  Include the installer in your application installer or updater.  
-1.  During your application installation or update, check if the Evergreen WebView2 Runtime is already installed on the user machine.  If not, the application invokes the installer to install the runtime.  
+1.  During your application installation or update, check if the Evergreen WebView2 Runtime is already installed on the user machine using the `GetAvailableCoreWebView2BrowserVersionString` API (or the .NET or WinUI equivalent).  
+1.  If not installed, the application installer/updater can silently invoke the runtime installer from an elevated process/command prompt with `MicrosoftEdgeWebView2RuntimeInstallerX64.exe /silent /install`. 
```

Comment:
```suggestion
1.  During your application installation or update, test to determine if the WebView2 Runtime (Evergreen) is already installed using [GetAvailableCoreWebView2BrowserVersionString](https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/0-9-538/webview2-idl#getavailablecorewebview2browserversionstring). If `nullptr` is returned, the application installer or application updater may silently install the runtime at an elevated command prompt using `MicrosoftEdgeWebView2RuntimeInstallerX64.exe /silent /install`. 
```

Reply:
Can you include the link to the ref docs for .Net and WinUI for the equivalents of GetAvailableCoreWebView2BrowserVersionString? I couldn't find them.

Reply:
Can we add a code snippet to show how to test using GetAvailableCoreWebView2BrowserVersionString? This would be really helpful.

Reply:
updated the api link and the return info.

Reply:
looks like some of the changes wasn't applied here.

Reply:
Didn't club the steps together and remove `process` from `elevated process/command prompt`. Do we need to do that?